rest-service/manager_rest/resource_manager.py

```diff
@@ -2640,6 +2699,142 @@ def _get_utc_datetime_from_sched_plan(time_expression, base_datetime=None):
             return parse_utc_datetime_relative(time_expression, base_datetime)
         return datetime.strptime(time_expression, time_fmt)
 
+    def _reset_sub_deployments(self, deployment_ids):
+        """Set sub-counts to 0 and sub-statuses to None, for deployment_ids.
+
+        We know that deployment_ids have no children, so we can clear their
```

Comment:
I assume this is intended to say that "This should only be called on deployment_ids that we know to have no children"?